pkg/scheduler/cache/interface.go

```diff
@@ -99,7 +99,7 @@ type VolumeBinder interface {
 
 // Binder interface for binding task and hostname
 type Binder interface {
-	Bind(kubeClient kubernetes.Interface, tasks []*api.TaskInfo) ([]*api.TaskInfo, []error)
```

Comment:
@wangyang0616 Do we have to change the interface? I am afraid of the compatibility.
Is there any better way to solve this problem?

Reply:
This interface was first broken in https://github.com/volcano-sh/volcano/pull/3547/files.